src/widget/wtracktableview.h

```diff
@@ -110,12 +110,19 @@ class WTrackTableView : public WLibraryTableView {
     QColor getTrackMissingColor() const {
         return m_trackMissingColor;
     }
+    // Default color for the track drop indicator line: red.
+    static constexpr const char* kDefaultDropIndicatorColor = "#ff0000";
```

Comment:
Is there value to store this constant as a string instead of a QColor directly?

Reply:
>  We use hex colors in skins so I opted for that.

Right, but you can just do `QColor(0xff, 0x00, 0x00)`... Thats basically the same as `QColor("#ff0000")`...